ui-gateway: Definition missing for Service 'adempiere.report.engine'

marcalwestf opened the pull request:
Fixes #116 

With the changes, it is now possible to run the project with script _start-all.sh_ using the service files.

@EdwinBetanc0urt  Edwin, it maybe of interest to you.

marcalwestf committed:
Add missing 'adempiere.report.engine' service definition to Docker Compose service file 10c-grpc_proxy_service_standard.yml

EdwinBetanc0urt commented:
Hi @marcalwestf Mario, it has not yet been placed to the standard because this service is under development, however having it there does not affect the current operation until the changes of vue in the `report-engine` branch are integrated to the main branch that would make use of those end-points of the report-engine service.

EdwinBetanc0urt commented:
I think the report engine grpc service should be separated in another .yml file, as is the processors grpc and the vue backend grpc.

marcalwestf committed:
add report engine service as a separate service for non-legacy files

marcalwestf commented:
@EdwinBetanc0urt  as you suggested, I implemented the service _adempiere.report.engine_ as a separate file.

In order to preserve the file sequence and to keep the script file _start-all.sh_ readable, I renamed the files following the report engine service.

I executed a docker compose dry run for standard stack legacy and non-legacy and the results were as expected.
